styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,558 @@
+      &:hover {
+        background: @surface0 !important;
+        color: @text !important;
+      }
+    }
+    .btn-success {
+      &,
+      &:hover {
+        color: @green !important;
+      }
+      &#submitButton {
+        background: @base !important;
+        &:hover {
+          background: @surface0 !important;
+        }
+      }
+    }
+    .btn-danger {
+      &,
+      &:hover {
+        color: @red !important;
+      }
+    }
+    .clipboard {
+      color: @text;
+      background: none;
+    }
+    #progressBar {
+      background-color: @blue !important;
+    }
+    .full-contrast:not(.active) path {
+      fill: @overlay2 !important;
+    }
+    .full-contrast.active path {
+      fill: @peach !important;
+    }
+    .edit-code {
+      background: @mantle;
+      color: @text !important;
+    }
+    .cb-tooltiptext {
+      color: @text !important;
+      background: @surface0 !important;
+    }
+    .cb-tooltiptext-bottom::after {
+      border-color: transparent transparent @surface0;
+    }
+    .headerlink {
+      color: @overlay1 !important;
+    }
+    #submitButton {
+      background: @mantle !important;
+    }
+    // claim-guide page
+    .color-change {
+      &[style="color:grey"] {
+        color: @overlay2 !important;
+      }
+      &[style="color:#039205"] {
+        color: @green !important;
+      }
+    }
+    // delete modal
+    .modal-content {
+      background: @crust;
+      color: @text;
+    }
+    .modal-body {
+      color: @overlay2;
+    }
+    .modal-header {
+      color: @text;
+    }
+    .modal-header,
+    .modal-footer {
+      border-color: @surface1 !important;
+    }
+    #deleteButton {
+      color: @red !important;
+    }
+    // footer
+    a {
+      color: @blue;
+    }
+    a:hover {
+      color: @sky !important;
+    }
+    .text-muted {
+      color: @overlay2 !important;
+    }
+    .text-primary {
+      color: @blue !important;
+    }
+    .text-success {
+      color: @green !important;
+    }
+    .text-warning {
+      color: @yellow !important;
+    }
+    .text-danger {
+      color: @red !important;
+    }
+    // editor
+    .CodeMirror-line::selection,
+    .CodeMirror-line > span::selection,
+    .CodeMirror-line > span > span::selection {
+      background-color: fade(@overlay2, 25%);
+    }
+    .cm-formatting-admonition {
+      color: @yellow;
+    }
+    .cm-mark {
+      color: @yellow;
+    }
+    .cm-link {
+      color: @blue;
+    }
+    .cm-url {
+      color: @sky;
+    }
+    .cm-formatting-toc {
+      color: @mauve;
+    }
+    .cm-variable-2 {
+      color: @green;
+    }
+    .cm-keyword {
+      color: @pink;
+    }
+    .cm-meta {
+      color: @subtext0;
+    }
+    .cm-comment {
+      color: @pink !important;
+    }
+    .cm-hr {
+      color: @surface2;
+    }
+    // markdown
+    .cm-header {
+      color: @text;
+    }
+    .cm-quote,
+    blockquote {
+      color: @green;
+      border-color: @green;
+    }
```

Comment:
I believe all these `.cm-*` token classes need `!important` for the colors because they aren't showing up in the editor properly, the default CSS has more specificity atm.

Reply:
hmm, they didn't need `!important` before. but i added it now (https://github.com/catppuccin/userstyles/pull/1227/commits/9667827fd0c56397825eb8eb8bfda77b8363add9).